silence/silence_test.go

```diff
@@ -523,8 +520,44 @@ func TestSilenceLimits(t *testing.T) {
 	// Do not check the exact size as it can change between consecutive runs
 	// due to padding.
 	require.Contains(t, err.Error(), "silence exceeded maximum size")
-	// TODO: Once we fix https://github.com/prometheus/alertmanager/issues/3878 this should be require.Equal.
```

Comment:
I removed this comment as making this transactional is proving more difficult then I expected.